gapic/src/main/com/google/gapid/views/TracerDialog.java

```diff
@@ -368,6 +370,15 @@ protected void configureDialog(DirectoryDialog dialog) {
         pcsWarning.setForeground(getDisplay().getSystemColor(SWT.COLOR_DARK_YELLOW));
         pcsWarning.setVisible(!models.settings.traceDisablePcs);
 
+        createLabel(this, "");
+        Link adbWarning = withLayoutData(
+            createLink(this,
+                "Path to adb invalid/missing. Please fix it in the <a>preferences</a>.",
```

Comment:
This suggests that adb is required, when for desktop usage it is not. Maybe the message should be something like:
`Path to adb invalid or missing, Android support disabled`

Reply:
Maybe we could put it in the device dropdown?
That way it only shows up when you try to do anything Android. (Not sure if its easy to jam stuff like this into the device dropdown)

Reply:
I've reworded the message.